### ISSUE TITLE
 overfitting and low accuracy 

### ISSUE THREAD

**Issue by Vishal321kumar** [SaiHitesh16/Glaucoma-Detection-using-CNN#3]
#1 pls help with project 
![Screenshot (102)](https://github.com/SaiHitesh16/Glaucoma-Detection-using-CNN/assets/92580370/d5efa3ef-76d6-4cc9-84d7-951bb9ed5553)
![Screenshot (104)](https://github.com/SaiHitesh16/Glaucoma-Detection-using-CNN/assets/92580370/811a2329-698d-4414-b0c1-ec3cf17549f2)
![Screenshot (105)](https://github.com/SaiHitesh16/Glaucoma-Detection-using-CNN/assets/92580370/1443ed70-1114-4365-912f-9604ae530d3d)
@SaiHitesh16

**Comment by SaiHitesh16** [comment]
How many datasets are u using? or How many training, testing, validation images are used for training and testing model?

**Comment by Vishal321kumar** [comment]
same as u 3 datasets acrima ,dhristi  and rim one a total of 1289 images

**Comment by Vishal321kumar** [comment]
i can give u my google colab link if u have time to look it up

**Comment by SaiHitesh16** [comment]
ok, tomorrow I will try to check

**Comment by Vishal321kumar** [comment]
https://colab.research.google.com/drive/1ie_szjsEtYnr7P7asf-vweQ785MAAgKj?usp=sharing

**Comment by Vishal321kumar** [comment]
pls take a look as soon as possible ,we doing this for final year project

**Comment by SaiHitesh16** [comment]
Try changing split ratio, re-run the epochs code to check if there is any change and try to decrease the epochs value because training accuracy is 1.00. I will check but try checking these bro .

**Comment by SaiHitesh16** [comment]
I couldn't access colab link,make it as accessible for everyone one bro.

**Comment by Vishal321kumar** [comment]
can i have ur linkdin email or anything to contact

**Comment by SaiHitesh16** [comment]
email :- 19kt1a0506@pscmr.ac.in

**Comment by Vishal321kumar** [comment]
shared access to that link

**Comment by YUGANDHAR2003** [comment]
same bro low accuracy and overfitting
help please?

**Comment by YUGANDHAR2003** [comment]
> https://colab.research.google.com/drive/1ie_szjsEtYnr7P7asf-vweQ785MAAgKj?usp=sharing

bro did you get the answer?

**Comment by Vishal321kumar** [comment]
> > https://colab.research.google.com/drive/1ie_szjsEtYnr7P7asf-vweQ785MAAgKj?usp=sharing
> 
> bro did you get the answer?

nah tried other things ,even changed the model still same ,  
if u have better accuracy more than (63%) pls share with me too

**Comment by SaiHitesh16** [comment]
Hi Yugandhar and Vishal, there might be issue in colab since it only assign weights so u have to wait till u get better accurate model and save it for further purpose

**Comment by YUGANDHAR2003** [comment]
if we use the same code why are we getting different accuracies? 
and it is overfitting and not working properly on unseen data

**Comment by YUGANDHAR2003** [comment]
and can i know how did you deploy the model on to streamlit

**Comment by YUGANDHAR2003** [comment]
hello Hitesh i really want to communicate  with you i need your help! it would be really great if you could really speak with me . Thank you!

**Comment by Vishal321kumar** [comment]
> hello Hitesh i really want to communicate with you i need your help! it would be really great if you could really speak with me . Thank you!

let me in the meet if u guys do it

**Comment by SaiHitesh16** [comment]
> if we use the same code why are we getting different accuracies? and it is overfitting and not working properly on unseen data

Based colab instance. Even I hadn't got this accuracy 1st time it took few days..
And Deploying in Streamlit is add ur streamlit code, ur cnn model in GitHub and deploy in Streamlit

**Comment by YUGANDHAR2003** [comment]
ok bro thank you. 
it would be really great if you could share the .h5 model ?

**Comment by SaiHitesh16** [comment]
check your mail, I have shared file link

**Comment by SaiHitesh16** [comment]
Issue is resolved so closing

**Comment by Vishal321kumar** [comment]
sorry to reopen the  chat i want .h5 model pls do share with me guys 
Your help is appreciated 
email - ironultimate321@gmail.com
